portal/templates/utilisation-report-service/utilisation-report-upload/check-the-report.njk

```diff
@@ -0,0 +1,102 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% block pageTitle %}Utilisation Report Upload{% endblock %}
+{% block content %}
+    {{ govukBackLink({
+        text: "Back",
+        href: "#"
+        }) }}
+    <h1 class="govuk-heading-l" data-cy="check-the-report-title">
+        Check the report
+    </h1>
+    {% if errorSummary %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errorSummary,
+          attributes: {
+            'data-cy': 'error-summary'
+          }
+        }) }}
+    {% endif %}
+    {% if validationErrors %}
+        <div class="govuk-form-group govuk-form-group--error govuk-!-margin-top-7">
+            <h2 class="govuk-heading-m" id="validation-errors-table">Errors found in {{ filename }}</h2>
+            <p class="govuk-error-message">
+                <span class="govuk-visually-hidden">Error:</span>
+                You must correct these errors before you can upload the report
+            </p>
+            <table class="govuk-table" data-cy="validation-errors-table">
+                <caption class="govuk-table__caption govuk-table__caption--m"></caption>
+                <thead class="govuk-table__head">
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header">Exporter</th>
+                        <th scope="col" class="govuk-table__header">Row</th>
+                        <th scope="col" class="govuk-table__header">Column</th>
+                        <th scope="col" class="govuk-table__header">Entry</th>
+                        <th scope="col" class="govuk-table__header">Error</th>
+                    </tr>
+                </thead>
+                <tbody class="govuk-table__body">
+                    {% for item in validationErrors %}
+                        <tr class="govuk-table__row" data-cy="validation-errors-table-row">
+                            <td class="govuk-table__cell">
+                                {% if item.exporter !== null %}
+                                    {{ item.exporter }}
+                                {% else %}
+                                    -
+                                {% endif %}
+                            </td>
+                            <td class="govuk-table__cell">
+                                {% if item.row !== null %}
+                                    {{ item.row }}
+                                {% else %}
+                                    -
+                                {% endif %}
+                            </td>
+                            <td class="govuk-table__cell">
+                                {% if item.column !== null %}
+                                    {{ item.column }}
+                                {% else %}
+                                    -
+                                {% endif %}
```

Comment:
Are `item.exporter`, `item.row`, `item.column` never undefined? Could/should we use e.g. `not item.exporter` instead?

Reply:
They can be undefined depending on the type of error, I tried using `item.column | Default('-')` but was having no luck with it